testFramework/com/twitter/intellij/pants/testFramework/OSSPantsIntegrationTest.java

```diff
@@ -63,7 +63,7 @@ protected void assertEmptyBeforeRunTask(RunConfiguration configuration) {
    * Assert Project has the right JDK and language level (JVM project only).
    */
   protected void assertProjectJdkAndLanguageLevel() {
-    final String pantsExecutablePath = PantsUtil.findPantsExecutable(getParentPath()).get().getPath();
+    final String pantsExecutablePath = PantsUtil.findPantsExecutable(getProjectPath()).get().getPath();
```

Comment:
This used the pants inside of `pants-intellij-plugin` instead of the one from the project. Export was failing due to:
```
FAILURE: No bootstrap callback registered for scalac in scala
```
It still fails if I run `./pants export` from the intellij plugin pants, any idea what it's about?

Reply:
Unclear. But thanks for fixing this!